src/app_service/service/wallet_account/async_tasks.nim

```diff
@@ -90,7 +90,7 @@ proc prepareTokensTask(argEncoded: string) {.gcsafe, nimcall.} =
     "storeResult": false
   }
   try:
-    let response = backend.fetchOrGetCachedWalletBalances(arg.accounts)
+    let response = backend.fetchOrGetCachedWalletBalances(arg.accounts, arg.storeResult)
```

Comment:
hmm I tried following all calls to this and it's not super clear to me the storeResult value means we need to force-refresh all balances. Until now we were relying on the backend logic to determine the cache lifetime, now we're basically refetching the values every single time.

Reply:
I don't actually understand why we wouldn't want to store whatever values we get from status-go in the local cache 🥴

Reply:
I'd keep the old logic (so `forceRefetch = false`) until we have a good reason to change it

Reply:
Me nighter, I could hardcode it to `false` in the call and that would be the same as it was before this change.
But I see no point in `storeResult` on the desktop app side without providing it to statusgo. That's why I included it in the call.

Maybe @Khushboo-dev-cpp may say more since she introduced it.

Reply:
@saledjenic was not introduced by me, but existed before, I might have moved the code around, I think its a good time to check if we can remove it